Hazelcast.Examples/App.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
```

Comment:
We should update the copyright in a separate PR.

Reply:
It has been sent as this PR:
https://github.com/hazelcast/hazelcast-csharp-client/pull/226

Reply:
Please remove this change from this PR

Reply:
Changed back to 2018